Src/ILGPU/Backends/PTX/PTXInstructions.Data.cs

```diff
@@ -543,6 +594,7 @@ partial class PTXInstructions
                 { (BinaryArithmeticKind.Min, ArithmeticBasicValueType.UInt16), "min.u16" },
                 { (BinaryArithmeticKind.Min, ArithmeticBasicValueType.UInt32), "min.u32" },
                 { (BinaryArithmeticKind.Min, ArithmeticBasicValueType.UInt64), "min.u64" },
+                { (BinaryArithmeticKind.Min, ArithmeticBasicValueType.Float16), "min.f16" },
```

Comment:
I've added the affected instructions to the file `PTXIntrinsics.Generated.tt`. Hence, ILGPU should not emit them since they don't have a proper implementation on targets with `SM <= SM_80`.

Reply:
Probably for a different PR, but should `MaxArchitecture` be exclusive? This might make it easier to define `SM < SM_80` rather than `SM <= SM_75`.

Reply:
That makes sense to me. This should avoid programming issues in the future. Let's address this in another "polishing" PR.